standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -8349,8 +8353,12 @@ public long get_latest_txnid_in_conflict(long txnId) throws MetaException {
 
   @Override
   public void commit_txn(CommitTxnRequest rqst) throws TException {
+    boolean isReplayedReplTxn = rqst.isSetReplPolicy() && rqst.isSetTxn_type() &&
```

Comment:
Why do you need this check - rqst.isSetTxn_type()?

Reply:
Will remove it.